tests/e2e/MediaCo/portal.spec.js

```diff
@@ -110,25 +109,15 @@ test.describe('E2E test', () => {
     const filePath = path.join(__dirname, '../../../src/assets/cableinfo.jpg');
     const attachInputId = await page.locator('div[id="attachment-container"] >> input').getAttribute('id');
     await page.setInputFiles(`#${attachInputId}`, filePath);
-
-    await Promise.all([
-      page.waitForResponse(
-        `${endpoints.serverConfig.infinityRestServerUrl}${
-          endpoints.serverConfig.appAlias ? `/app/${endpoints.serverConfig.appAlias}` : ''
-        }/api/application/v2/attachments/upload`
-      )
-    ]);
+    await page.waitForTimeout(5000);
```

Comment:
why are you having hard coded wait timeout instead of wait until api completion?